packages/EasyStandard/src/Sniffs/Arrays/OrderArrayKeysAlphabeticallySniff.php

```diff
@@ -0,0 +1,253 @@
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+final class OrderArrayKeysAlphabeticallySniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    public const ARRAY_KEYS_SORT_ALPHABETICALLY = 'ArrayKeysSortAlphabetically';
+
+    /**
+     * @var string
+     */
+    private const COMMENT_CONTENT = '[comment]';
+
+    /**
+     * @var mixed[]
+     */
+    public $skipPatterns = [];
+
+    /**
+     * @param int $bracketOpenerPointer
+     */
+    public function process(File $phpcsFile, $bracketOpenerPointer): void
+    {
+        if ($this->shouldSkip($phpcsFile, $bracketOpenerPointer)) {
+            return;
+        }
+
+        $tokens = $phpcsFile->getTokens();
+        $token = $tokens[$bracketOpenerPointer];
+        $bracketCloserPointer = $token['bracket_closer'] ?? $token['parenthesis_closer'];
+        $code = $phpcsFile->getTokensAsString($bracketOpenerPointer, $bracketCloserPointer - $bracketOpenerPointer + 1);
+
+        $parser = (new ParserFactory())->create(ParserFactory::PREFER_PHP7);
+        try {
+            $ast = $parser->parse('<?php' . PHP_EOL . $code . ';');
+        } catch (Error $error) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->addErrorOnLine(
+                "Parse error: {$error->getMessage()}",
+                $token['line'],
+                self::ARRAY_KEYS_SORT_ALPHABETICALLY
+            );
+            $phpcsFile->fixer->endChangeset();
+
+            return;
+        }
+
+        if ($ast === null) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->addErrorOnLine(
+                'Unknown error during parse code',
+                $token['line'],
+                self::ARRAY_KEYS_SORT_ALPHABETICALLY
+            );
+            $phpcsFile->fixer->endChangeset();
+
+            return;
+        }
+
+        /** @var \PhpParser\Node\Stmt\Expression $stmtExpr */
+        $stmtExpr = $ast[0];
+        /** @var \PhpParser\Node\Expr\Array_ $array */
+        $array = $stmtExpr->expr;
+
+        $array = $this->refactor($array);
+
+        if ($array === null) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->addErrorOnLine(
+                'Unknown error during refactor array',
+                $token['line'],
+                self::ARRAY_KEYS_SORT_ALPHABETICALLY
+            );
+            $phpcsFile->fixer->endChangeset();
+
+            return;
+        }
+
+        if ($array !== null && $array->hasAttribute('isChanged') === false) {
+            return;
+        }
+
+        $prettyPrinter = new Standard();
+        $newContent = $prettyPrinter->prettyPrint([$array]);
+        $newContent = \str_replace('    ' . self::COMMENT_CONTENT . PHP_EOL, '', $newContent);
+
+        $phpcsFile->fixer->beginChangeset();
+
+        $fix = $phpcsFile->addFixableError(
+            'Array\'s keys should be sorted alphabetically',
```

Comment:
We don't need `\'s` here: `The array keys should be sorted alphabetically`. Here and below.

Reply:
Done.